en/dev_log/logging.md

```diff
@@ -100,33 +100,27 @@ amount of dropouts:
 
 ## SD Cards
 
-The following provides performance results for different SD cards.
-Tests were done on a Pixracer; the results are applicable to Pixhawk as well.
-
-:::note
 The maximum supported SD card size for NuttX is 32GB (SD Memory Card Specifications Version 2.0).
-:::
+The **SanDisk Extreme U3 32GB** is recommended as it does not exhibit write-time spikes (and thus virtually no dropouts).
```

Comment:
We also use the [Samsung EVO Plus 32](https://www.samsung.com/ch/memory-storage/memory-card/evo-plus-microsd-card-64gb-mb-mc64ha-eu/) successfully.